integrations/webchat/examples/speech-and-text/DOCS.md

```diff
@@ -0,0 +1,103 @@
+This will cover how to extend Watson Assistant web chat to be able to use the Watson Speech service to convert speech to text and text to speech.
```

Comment:
`Watson Speech` as link to watson speech marketing page?

Reply:
The irony here is that may include a link to the SDK here. I did actually open the SDK documentation and it does not have any links to Watson Speech so I wasn't entirely sure what link to use. But I can dig up some links from Google.